libs/npm/core/src/components/tool-bar/styled/ToolBar.tsx

```diff
@@ -8,7 +8,6 @@ export const ToolBar = styled(Row)<IToolBarProps>`
     props.variant === 'dark'
       ? props.theme.css.darkerBackgroundColor
       : props.theme.css.backgroundColor};
-  max-height: 10.95em;
```

Comment:
@jdtoombs Was there a reason to limit the height?  It's a horrible user experience.  It's much better now that it's removed.